ruby/vampires.rb

```diff
@@ -0,0 +1,77 @@
+puts "How many employees are being interviewed"
+interviewed = gets.to_i
+
+while interviewed >= 1
+
+puts "What is your name?"
+name = gets.chomp
+
+puts "How old are you?"
+age = gets.to_i
+
+puts "What year were you born?(yyyy)"
+year = gets.to_i
+
+puts "Would you like garlic bread from the cafeteria?(yes, no)"
+garlic_preference = gets.chomp
+
+puts "Would you like to enroll in the health insurance?(yes, no)"
+insurance = gets.chomp
+
+  if 2017 - year == age
+    right_age = true
+  else
+    right_age = false
+  end
+
+
+  if garlic_preference == "yes"
```

Comment:
same refactor as above, no need for an if/else if all you are trying to do is set a variable to a boolean.